features/tracker-allowlist.json

```diff
@@ -296,6 +296,17 @@
                     }
                 ]
             },
+            "segment.io": {
+                "rules": [
+                    {
+                        "rule": "cdn.segment.com/analytics.js/v1/4I2fAtE5IxTlczNT3F91OckHtI5wfR7g/analytics.min.js",
+                        "domains": [
+                            "foxnews.com",
```

Comment:
You have a hanging comma here which is causing the validation error.

Reply:
@SlayterDev sorry - I removed the comma.